inference-engine/tests/ie_test_utils/common_test_utils/xml_net_builder/ir_net.hpp

```diff
@@ -398,13 +398,25 @@ class IRBuilder_v10 {
         return m_ir_net->serialize();
     }
 
+protected:
+    IRBuilder_v10(const std::string &name, int version) :
+            m_ir_net(IRNet::Create({{"name",    name},
+                                    {"version", std::to_string(version)}})) {
+    }
+
 private:
     size_t m_offset = 0;  // for Constant nodes
 
     std::string m_latest_layer_name;
     std::shared_ptr<IRNet> m_ir_net;
 };
 
+///////////////////////////////// IR v7 ///////////////////////////////
+class IRBuilder_v7 : public IRBuilder_v10 {
+public:
+    explicit IRBuilder_v7(const std::string &name) : IRBuilder_v10(name, 7) {}
+};
```

Comment:
why do we need it? IR v7 seems deprecated

Reply:
V10 had no "Memory" representation until recently. And it was not introduced in master when we start write these test cases. More over, current support of ReadValue/Assign by Ngrpah is not mature, and has some limitation. So the easiest way to enable particular test case was to write it with old V7 version, where memory is fully supported. This patch introduce 3 test new test case:
1. Functional test with "ciclic buffer" - developed by me. Based on ngraph.  
2. Functional test on bf16 transformer. - developed by Yury Gaydaychuck. Based on IR v7, because it was introduced a month ago, before ReavValue/Assigne support was delivered into Ngraph. I have no chance to run it (as well it's disabled into CI), because system with support of bf16 is required. I decided to keep it as is. 
3. Unit test for bf16 transformer (#2 equivalent) - developed by me. Migrated to Ngraph.

If you wish I can just totally remove test introduced by Yury.